R/generate_rules.R

```diff
@@ -2,8 +2,9 @@
 #' Generate rules
 #'
 #' @description
-#' Generates a set of relevant decision rules characterizing the
-#' heterogeneity in the ITE.
+#' Generate a list of Rules characterizing the heterogeneity in the Conditional
```

Comment:
Generate -> Generates
There is a difference between R and Python styling convention. 
Rules -> rules

Reply:
Got it. Thank you for the feedback. I was not aware of this convention.